EmailageJavaScriptWrapper/EmailValidator.js

```diff
@@ -0,0 +1,140 @@
+function emailValidate(requestObject, accountSId, authToken, baseUrl, requestType, dataType, sigMethod, nonce, version) {
+    if (baseUrl == undefined || (baseUrl.length) > 0 == false) {
+        baseUrl = "https://api.emailage.com/EmailAgeValidator/"
+    }
+
+    if (!sigMethod)
+        sigMethod = "HMAC-SHA1";
+
+    if (!version)
+        version = "1.0";
+
+    var authObject = getAuthObject(dataType, accountSId, sigMethod, version);
+
+    if (nonce)
+        authObject.oauth_nonce = nonce;
+
+    var oauthPostModel = getOAuthPostModel(requestType, authObject, baseUrl);
+    var oauthData = getOauthData(oauthPostModel);
+    var sig = oauthSign(authToken + '&', oauthData, sigMethod);
+    var requestUrl = oauthPostModel.action + "?" + getParameterString(oauthPostModel.parameters) + "&oauth_signature=" + percentEncode(sig);
+    var result = "";
+
+    $.ajax({
+        type: requestType,
+        url: requestUrl,
+        dataType: dataType,
+        data: requestObject,
+        async: false,
+        success: function (emailValidResult) {
+            return (emailValidResult);
+        },
+        error: function (xmlhttprequest, textstatus, message) {
+            return textstatus;
+        }
+    }).done(function (response) {
+        result = response;
+    });
+    return result;
+};
+
+var getAuthObject = function (dataType, accountSId, sigMethod, version) {
+
+    if (!sigMethod)
+        sigMethod = "HMAC-SHA1";
```

Comment:
sigMethod = "HMAC-SHA256"
???

Reply:
Setting this as a default, honestly that can be HMAC-SHA256, HMAC-SHA234,HMAC-SHA512. I'll add a select drop down for this in the sample form